SDK/AppCenter/Microsoft.AppCenter.WindowsDesktop/Microsoft.AppCenter.WindowsDesktop.csproj

```diff
@@ -24,6 +24,10 @@
     <PackageReference Include="System.Configuration.ConfigurationManager" Version="4.6.0" />
     <PackageReference Include="Microsoft.Windows.SDK.Contracts" Version="10.0.18362.2005" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'net5.0-windows'">
```

Comment:
Let's use `And` and unite `'net5.0-windows` and `net5.0-windows10.0.17763.0` under one condition, as they both reference the same versions of deps.